src/privatekey.cpp

```diff
@@ -284,6 +284,13 @@ std::vector<uint8_t> PrivateKey::Serialize(const bool fLegacy) const
     return data;
 }
 
+std::array<uint8_t, PrivateKey::PRIVATE_KEY_SIZE> PrivateKey::SerializeToArray(bool fLegacy) const
+{
+    std::array<uint8_t, PRIVATE_KEY_SIZE> data{};
+    Serialize(data.data());
```

Comment:
is it forgotten to use `fLegacy` here?

Reply:
Not sure, but I'm guessing no? both PrivateKey::Serialize(uint8_t *buffer) and PrivateKey::Serialize(const bool fLegacy) don't use it. Maybe. private keys are not affected by legacy / non legacy

Reply:
Yes, it's a dummy one here which is needed because our `CBLSWrapper` expects it to be that way for every `ImplType`.